presto-hive/src/main/java/com/facebook/presto/hive/statistics/MetastoreHiveStatisticsProvider.java

```diff
@@ -102,6 +175,67 @@ public TableStatistics getTableStatistics(ConnectorSession session, ConnectorTab
         return tableStatistics.build();
     }
 
+    private boolean isLowHighSupportedForType(Type type)
+    {
+        if (type instanceof DecimalType) {
+            return true;
+        }
+        if (type.equals(TINYINT)
```

Comment:
why not VARCHAR?  Is that a Hive limitation?  In the previous commit you have low/high values in show stats for varchar.

Reply:
@losipiuk ?

Reply:
Yeah. Hive does not return low and high value for character types. AFAIR they have some information on min/mean/max length of values, but we do not expose that via SPI yet.